packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -33,14 +33,20 @@ set timeDilation(double value) {
   _timeDilation = value;
 }
 
-/// A frame-related callback from the scheduler.
+/// Signature for frame-related callbacks from the scheduler.
 ///
 /// The timeStamp is the number of milliseconds since the beginning of the
 /// scheduler's epoch. Use timeStamp to determine how far to advance animation
 /// timelines so that all the animations in the system are synchronized to a
 /// common time base.
 typedef void FrameCallback(Duration timeStamp);
 
+/// Signature for [Scheduler.scheduleTask] callbacks.
+///
+/// The type argument is the task's return value. Consider [void] if the task
```

Comment:
nit: I'd say "The type argument `T` is the..."

Reply:
done